apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿#region
```

Comment:
resharper does this as well as autoformat code.
How about i follow up with a change that does a full code cleanup pass with modified resharper settings that fit the team's expectations?  There are other parts of the code that have whitespace and other similar issues and if I do this here, its going to explode the change and it wont be easy to tell the difference between a resharper code cleanup change and a change I made.